src/components/NotificationsForm.js

```diff
@@ -157,7 +197,18 @@ class NotificationsForm extends React.Component {
                         </Typography>
 
                         <FormBuilder
-                            fields={this.getFields()}
+                            fields={this.getFirstSectionFields()}
+                            onUpdateField={this.onUpdateField}
+                        />
+                        <Typography gutterBottom variant="h6" component="h3">
```

Comment:
Can you please add some more top margin here so the section title doesn't look that close to the Phone field?